### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by alertingjames** [AgoraIO/OpenVideoCall-Android#26]
D:\Sample\VideoCall\OpenVideoCall-Android-master\app\src\main\cpp\video_preprocessing_plugin_jni.cpp
Error:(5, 10) fatal error: 'agora/IAgoraRtcEngine.h' file not found
Error:(57, 43) error: no member named 'AGORA_IID_MEDIA_ENGINE' in namespace 'agora::rtc'; did you mean 'agora::AGORA_IID_MEDIA_ENGINE'?

**Comment by Mbanefo** [comment]
Indeed, alertingjames. Agora, your work is great. Thank you. However, the above issue has brought me here, too. How do we solve this issue?

**Comment by Mbanefo** [comment]
How do we locate this missing file?
